packages/components/src/inputs/tests/chromatic/NumberField.stories.tsx

```diff
@@ -0,0 +1,207 @@
+import { MailIcon } from "@hopper-ui/icons";
+import { Div } from "@hopper-ui/styled-system";
+import type { Meta, StoryObj } from "@storybook/react";
+import { within } from "@storybook/test";
+
+import { ErrorMessage } from "../../../ErrorMessage/index.ts";
+import { HelperMessage } from "../../../HelperMessage/index.ts";
+import { Inline, Stack } from "../../../layout/index.ts";
+import { Label } from "../../../typography/index.ts";
+import { NumberField, type NumberFieldProps } from "../../src/NumberField.tsx";
+
+const meta = {
+    title: "Components/NumberField",
+    component: NumberField
+} satisfies Meta<typeof NumberField>;
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+export const Default: Story = {
+    render: args => (
+        <Stack>
+            <Inline alignY="center">
+                <NumberField {...args} />
+                <NumberField size="sm" {...args} />
+            </Inline>
+            <NumberField minValue={0} maxValue={100} {...args} />
+            <NumberField isDisabled {...args} />
+            <NumberField isReadOnly {...args} />
+            <NumberField isFluid {...args} />
+            <Div width="10%">
+                <NumberField isFluid {...args} />
+            </Div>
+        </Stack>
+    ),
+    args: {
+        "aria-label": "Label"
+    }
+};
+
+export const WithLabel: Story = {
+    ...Default,
+    args: {
+        ...Default.args,
+        "aria-label": undefined,
```

Comment:
good question... it must have been like that for TextField as well... I'll look into it.

Reply:
alright so it's using the args from the Default story, which has aria-label defined instead of children. In this case, we're setting the children so we don't need aria-label.
You do bring up a point though, aria-label is the only thing set for the default children so it doesn't really make sense to even use the args in this case. I can remove it. Maybe Alex was just trying to be consistent?